src/getChildEventSubscriber.js

```diff
@@ -46,13 +32,12 @@ export default function getChildEventSubscriber(
     const subscribers = getChildSubscribers(payload.type);
     subscribers &&
       subscribers.forEach(subs => {
-        // $FlowFixMe - Payload should probably understand generic state type
         subs(payload);
       });
   };
 
-  let isSelfFocused = false;
-
+  let isParentFocused = true;
```

Comment:
can we clarify in a comment somewhere here what parent and child mean in this context, and why parent initializes as focused and child as unfocused?